subnets/config.md

```diff
@@ -12,9 +12,11 @@ Here is an example of Subnet config file:
 ```json
 {
   "validatorOnly": false,
-  "consensusParameters": {
-    "k": 25,
-    "alpha": 18
+  "consensusConfig": {
+    "snowballParameters": {
```

Comment:
I think we should call these new params, snowflake params and not snowball params. Because snowball is now no longer used, and also the k and alpha are more snowflake specific.